cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/passes/SymbolResolver.kt

```diff
@@ -992,6 +826,42 @@ open class SymbolResolver(ctx: TranslationContext) : ComponentPass(ctx) {
                 ?.createInferredConstructor(constructExpression.signature)
     }
 
+    fun tryFunctionInference(
+        call: CallExpression,
+        result: CallResolutionResult,
+    ): List<FunctionDeclaration> {
+        // We need to see, whether we have any suitable base (e.g. a class) or not; but only if
```

Comment:
Does this comment actually reflect the code? "but only if the call itself is not already scoped" is not immediately visible to me here

Reply:
It's the `!call.callee.name.isQualified()` part

Reply:
Ok you are correct, the comment is wrong or misleading. I will change the comment. This behaviour will anyway be redesigned as part of https://github.com/Fraunhofer-AISEC/cpg/pull/1586